mackup/applications/starship.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = Starship
+
+[configuration_files]
```

Comment:
This should be moved under `xdg_configuration_files`, see https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory

Reply:
Does this still apply when the application consuming the config file doesn't look for `XDG_CONFIG_HOME`, but hardcodes `.config` as the path it reads from? That's what Starship's doing here: https://github.com/starship/starship/blob/1eabd527252291398df5749d30f3459ea2a03823/src/config.rs#L184

Reply:
heh, no it doesn't.
Weird.